src/tools/genpolicy/rules.rego

```diff
@@ -1203,8 +1203,15 @@ match_caps(p_caps, i_caps) if {
     count(p_caps) == 1
     p_caps[0] == "$(default_caps)"
 
+    print("match_caps 2: i_caps =", i_caps)
     print("match_caps 2: default_caps =", policy_data.common.default_caps)
-    policy_data.common.default_caps == i_caps
+
+    norm_defaults := { strip_cap_prefix(c) | c := policy_data.common.default_caps[_] }
+    norm_input := { strip_cap_prefix(c) | c := i_caps[_] }
+    print("match_caps 2: norm_defaults =", norm_defaults)
```

Comment:
I would prefer to keep the prints from "match caps 2" consistent with the prints from "match caps 3" - i.e., either:
- Add these new prints into "match caps 3" too, or
- Remove these new prints from "match caps 2".